test/unit/Query/QueryBuilder.php

```diff
@@ -84,11 +84,11 @@ public function testAddFilter()
             )
             ->then
             ->array($this->testedInstance->getQueryBody())
-                ->array['query']->array['bool']->array['must']->object['match_all']->isEqualTo((object)[])
+                ->array['query']->array['filtered']->array['query']->array['bool']->array[CombiningFactor::MUST]->array['match_all']->isEqualTo([])
             ->array($this->testedInstance->getQueryBody())
-                ->array['query']->array['bool']->array['filter']->array[0]->isEqualTo(['term' => ['size' => 'M']])
+                ->array['query']->array['filtered']->array['filter']->array[0]->isEqualTo(['term' => ['size' => 'M']])
             ->array($this->testedInstance->getQueryBody())
-                ->array['query']->array['bool']->array['filter']->array[1]->isEqualTo(['term' => ['color' => 'blue']])
+                ->array['query']->array['filtered']->array['filter']->array[1]->isEqualTo(['term' => ['color' => 'blue']])
```

Comment:
This set of changes is failing because there is no `'filtered'` key in the array (see https://github.com/novaway/elasticsearch-client/pull/11/files#diff-1d8c055988687409416e6274c180cd28L148)